sdk/metricsadvisor/ai-metrics-advisor/sample.env

```diff
@@ -3,6 +3,13 @@
 METRICS_ADVISOR_ENDPOINT=https://<resource name>.cognitiveservices.azure.com/
 METRICS_ADVISOR_SUBSCRIPTION_KEY=<subscription key>
 METRICS_ADVISOR_API_KEY=<api key>
+# METRICS_ADVISOR_METRIC_ID=
+# METRICS_ADVISOR_DETECTION_CONFIG_ID=
+# METRICS_ADVISOR_HOOK_ID=
+# METRICS_ADVISOR_DATAFEED_ID=
+# METRICS_ADVISOR_INCIDENT_ID=
+# METRICS_ADVISOR_SQL_SERVER_CONNECTION_STRING=
+# METRICS_ADVISOR_AZURE_SQL_SERVER_QUERY=
```

Comment:
What's this?

Reply:
These are sample env variables required for samples and tests